<launch>
  <arg name="prefix"		default="ftsensor_"/>
  <arg name="dev"		default="/dev/ttyUSB0"/>
  <arg name="publish_rate"	default="125"/>

  <node name="$(arg prefix)driver"
        pkg="aist_ftsensor" type="dynpick_driver" output="screen">
    <rosparam file="$(dirname)/../config/$(arg prefix)driver.yaml"
	      subst_value="true"/>
  </node>

  <!-- <rosparam file="$(dirname)/../config/force_torque_sensor_controller.yaml" -->
  <!-- 	    subst_value="true"/> -->
  <!-- <node name="controller_spawner" -->
  <!--       pkg="controller_manager" type="spawner" output="screen" -->
  <!-- 	args="force_torque_sensor_controller"/> -->

  <rosparam file="$(dirname)/../config/aist_ftsensor_controller.yaml"
  	    subst_value="true"/>
  <node name="controller_spawner"
        pkg="controller_manager" type="spawner" output="screen"
  	args="aist_ftsensor_controller"/>

</launch>
